packages/-ember-data/ember-cli-build.js

```diff
@@ -4,7 +4,17 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function (defaults) {
   const isProd = process.env.EMBER_ENV === 'production';
+  const needsIE11 = !!process.env.TARGET_IE11;
```

Comment:
Does this come across as a string?

Reply:
if its defined at all it works, its the same check we do elsewhere. That said we should also ensure we are in the test environment because otherwise this will affect the asset size check build negatively.

Reply:
Oh right we never set it to `false`